ecosystem/sep-0031.md

```diff
@@ -216,21 +214,11 @@ earlier than the quote's expiration.
 ### Detailed Sending Anchor Flow
 
 1. The Sending Client initiates a payment to the Receiving Client.
-1. The Sending Anchor identifies the Receiving Anchor it will use for the payment based on the receipient's location.
+1. The Sending Anchor identifies the Receiving Anchor it will use for the payment based on the recipient's location.
 1. The Sending Anchor makes a request to the Receiving Anchor's `GET /info` endpoint to collect asset information and
    the `transaction.fields` describing the transaction-related information required by the Receiving Anchor.
-1. The Sending Anchor fetches the relevant KYC `type` values from the `sep12` object for the Sending and Receiving
-   Client. If no `type` values are defined for either client, KYC information is not required for that client.
-1. The Sending Anchor makes [SEP-12 GET /customer?type=](sep-0012.md#customer-get) requests for the Sending and
-   Receiving Clients. The response includes the [SEP-9](sep-0009.md) KYC attributes required for registering a client of
-   the associated `type`.
-1. The Sending Anchor collects all required information from the Sending Client. This includes the custom fields listed
-   in the Receiving Anchor's `GET /info` response as well as the KYC fields described in the `GET /customer` responses
-   for both the Sending and Receiving Clients. How the Sending Anchor collects this information from the Sending Client
-   is out of the scope of this document, but the Receving Client should not be required to take any action in order for
-   the Sending Client to make the payment to the Receiving Anchor.
-1. The Sending Anchor makes [SEP-12 PUT /customer](sep-0012.md#customer-put) requests containing the collected
-   information for each client.
+1. Optionally the Sending Anchor can pre-register clients using [SEP-12 PUT /customer](sep-0012.md#customer-put) and
```

Comment:
I think it'd be nice to have a bit more detailed description of how it works (basically explaining flow chart above)